### ISSUE TITLE
Memory Error

### ISSUE THREAD

**Issue by aqsasamreen** [ck090/Quick_Draw#2]
I am facing memory issue when train the system about 14 classes of images and i need to train the system for 345 classes of the image. Please suggest me something effective.

**Comment by ck090** [comment]
Ohh thats a lot ..!! try reducing the batch size of the training step.
But the accuracy will decrease. 345 classes of how many images in each class.

**Comment by aqsasamreen** [comment]
All classes of quick draw dataset.

**Comment by ck090** [comment]
Well than it's hard to train that many using a normal CPU, you might have to use a GPU enabled system that is going to be capable of training on that many classes.
Otherwise there is no other option but try something along the lines of Transfer learning!

**Comment by aqsasamreen** [comment]
But i need to train neural network on quickdraw dataset.

**Comment by ck090** [comment]
Sorry, did you try transfer learning!!

**Comment by aqsasamreen** [comment]
I have no idea about transfer learning.

**Comment by aqsasamreen** [comment]
I split up the data and train more than one models.And now i want to load
all models into single model for testing one new image but i am stuck and
unable to solve the problem.Could you please suggest me something useful.

**Comment by ck090** [comment]
What do you do or which college do you go to!
